src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs

```diff
@@ -476,8 +476,16 @@ private static List<MediaInfo> QueryMedia(SelectArguments arguments)
         /// <privilege>http://tizen.org/privilege/content.write</privilege>
         /// <param name="mediaId">The media ID to delete.</param>
         /// <returns>true if the matched record was found and deleted, otherwise false.</returns>
-        /// <remarks>The <see cref="MediaDatabase.ScanFile(string)"/> or the <see cref="MediaDatabase.ScanFolderAsync(string)"/> can be used instead.</remarks>
-        /// <exception cref="InvalidOperationException">The <see cref="MediaDatabase"/> is disconnected.</exception>
+        /// <remarks>
```

Comment:
And please add more description about returing false case

Reply:
I'll remove `handle.IsInvalid` case and so it's enough to describe false case with current description.